### PR TITLE
Added additonal Windows specific setup intructions and how to fix issues

### PR DESCRIPTION
**Pull request by rsanchez79**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -19,8 +19,16 @@ installed ruby (not system ruby) and a clean gemset dedicated to rspec-dev:
...
-
+### Windows Instructions
+Windows users can use [uru](https://bitbucket.org/jonforums/uru) to manage Ruby versions.
+Fork the following projects: 
+	[rspec](https://github.com/rspec/rspec)
+	[rspec-core](https://github.com/rspec/rspec-core)
+	[rspec-expectations](https://github.com/rspec/rspec-expectations)
+	[rspec-mocks](https://github.com/rspec/rspec-mocks)
+	[rspec-rails](https://github.com/rspec/rspec-rails) 
+	[rspec-support](https://github.com/rspec/rspec-support)
```
Comment (JonRowe) [inline README.markdown]: This is not really relevant to Windows, and in any case the point of the `rspec-dev` repo is to take care of this for a developer, (and `rspec-rails` is only necessary to work with Rails, and `rspec` itself isn't necessary at all).

File: README.markdown [hunk 2]
```diff
@@ -114,6 +122,11 @@ command from the root of the repository you're working on. i.e. if you're
 working on rspec-core, cd to the rspec-core directory. Don't try to run specs
 from the rspec-dev directory, or you'll see `LoadError`s.
 
+## Errors in Windows setup
+
+If you get a SSL error in Windows, you can follow the instructions on this [link](https://gist.github.com/luislavena/f064211759ee0f806c88) to fix it.
+If you get this error "Gem::InstallError: The 'redcarpet' native gem requires installed build tools", download the development kit from [http://rubyinstaller.org/downloads]. You can follow the installation instructions [here](https://github.com/oneclick/rubyinstaller/wiki/Development-Kit).
```
Comment (JonRowe) [inline README.markdown]: I'd like this hardwrapped if you wouldn't mind, and the error message could probably use some `'s to make it highlighted.

**Comment by JonRowe** [comment]
Thanks for the input :) We appreciate help from windows users :)

**Comment by rsanchez79** [comment]
I made multiple commits so I made a mess. Can we close this PR and send a new one with 1 commit only?

**Pull request closed without merging**